internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStoreGraveyard.java

```diff
@@ -156,7 +204,7 @@ public List<String> listStoreNamesFromGraveyard(String clusterName) {
    * @return  Matching store from each venice. Normally contains one element.
    * If the store existed in some other cluster before, there will be more than one element in the return value.
    */
-  private List<Store> getStoreFromAllClusters(String storeName) {
```

Comment:
Feel free to add `com.linkedin.venice.annotation.VisibleForTesting` annotation if visibility is changed for unit testing only